google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/slow/ITBackupTest.java

```diff
@@ -175,31 +174,40 @@ public void setUp() {
     logger.info("Finished checking existing test backup operations");
   }
 
-  @After
-  public void tearDown() throws Exception {
+  @AfterClass
+  public static void tearDown() throws Exception {
+    logger.info("Starting test teardown");
     for (String backup : backups) {
+      logger.info(String.format("Waiting for optimize operation for backup %s to finish", backup));
       waitForDbOperations(backup);
+      logger.info(String.format("Deleting backup %s", backup));
       dbAdminClient.deleteBackup(testHelper.getInstanceId().getInstance(), backup);
     }
     backups.clear();
     for (String db : databases) {
+      logger.info(String.format("Dropping database %s", db));
       dbAdminClient.dropDatabase(testHelper.getInstanceId().getInstance(), db);
     }
   }
 
-  private void waitForDbOperations(String backupId) throws InterruptedException {
+  private static void waitForDbOperations(String backupId) throws InterruptedException {
     try {
       Backup backupMetadata =
           dbAdminClient.getBackup(testHelper.getInstanceId().getInstance(), backupId);
       boolean allDbOpsDone = false;
       while (!allDbOpsDone) {
         allDbOpsDone = true;
+        assertNotNull(backupMetadata.getProto());
```

Comment:
Should this be pushed outside of the `while` loop, since we only update the `backupMetadata` in the beginning of the `try` block?